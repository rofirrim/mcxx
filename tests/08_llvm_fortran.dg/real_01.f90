! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="real_01.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    REAL :: A, B, C
    A = 10.0
    B = 4.0

    C = A + B
    PRINT *, C

    C = A - B
    PRINT *, C

    C = B - A
    PRINT *, C

    C = A * B
    PRINT *, C

    C = A / B
    PRINT *, C

    C = B / A
    PRINT *, C

    C = A ** B
    PRINT *, C

    C = B ** A
    PRINT *, C
END PROGRAM MAIN
