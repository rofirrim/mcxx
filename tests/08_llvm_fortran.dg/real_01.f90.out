   14.0000000    
   6.00000000    
  -6.00000000    
   40.0000000    
   2.50000000    
  0.400000006    
   10000.0000    
   1048576.00    
